lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if an example includes any expectation.
+      #
+      # All RSpec's example and expectation methods are covered by default.
+      # If you are using your own custom methods,
+      # add the following configuration:
+      #
+      #   RSpec/NoExpectationExample:
+      #     ExampleMethodnames:
+      #       - custom_it
+      #     ExpectationMethodNames:
+      #       - custom_expect
+      #
+      # @example
+      #
+      #   # bad
+      #   it do
+      #     a?
+      #   end
+      #
+      #   # good
+      #   it do
+      #     expect(a?).to be_truthy
+      #   end
+      class NoExpectationExample < Base
+        extend AutoCorrector
+
+        include RangeHelp
+
+        DEFAULT_EXAMPLE_METHOD_NAMES = ::Set[
```

Comment:
This should be already defined in the `Language` module. Perhaps check out the MultipleExpectation cop for an example how to count the expectations in an example